pkg/url/url.go

```diff
@@ -246,15 +248,48 @@ func ListPushed(client *occlient.Client, componentName string, applicationName s
 // ListPushedIngress lists the ingress URLs on cluster for the given component
 func ListPushedIngress(client *kclient.Client, componentName string) (URLList, error) {
 	labelSelector := fmt.Sprintf("%v=%v", componentlabels.ComponentLabel, componentName)
-	klog.V(4).Infof("Listing ingresses with label selector: %v", labelSelector)
-	ingresses, err := client.ListIngresses(labelSelector)
+	klog.V(4).Infof("Listing ingressesExtensionV1Beta1 with label selector: %v", labelSelector)
+	var ingressNetworkingV1Beta1 []networkingv1beta1.Ingress
+	var ingressNetworkingV1 []networkingv1.Ingress
+	klog.V(4).Infof("Listing extension v1beta1 ingressesExtensionV1Beta1")
+	ingressesExtensionV1Beta1, err := client.ListExtensionV1Beta1Ingresses(labelSelector)
 	if err != nil {
 		return URLList{}, errors.Wrap(err, "unable to list ingress names")
 	}
 
+	if supported, err1 := client.IsNetworkingV1Beta1IngressSupported(); err1 != nil {
+		if supported {
+			ingressNetworkingV1Beta1, err = client.ListNetworkingV1Beta1Ingress(labelSelector)
+			if err != nil {
+				return URLList{}, errors.Wrap(err, "unable to list networking v1 beta1 ingress names")
+			}
+		}
+	} else {
```

Comment:
shouldn't we try first if NetworkingV1 is supported or not before returning a blank url list?